.travis.sh

```diff
@@ -9,6 +9,6 @@ lint)
   rubocop --lint --display-style-guide --extra-details --display-cop-names
   ;;
 *)
-  KITCHEN_LOCAL_YAML='.kitchen.docker.yml' kitchen test "$SUITE" --concurrency=2
+  KITCHEN_LOCAL_YAML='.kitchen.docker.yml' kitchen test "$SUITE" --concurrency=2 --log-level=debug
```

Comment:
i was planning to remove it, but the logs in travis arent too big with it and it was really useful for debugging a problem with kitchen-dokken so I vote to leave it in